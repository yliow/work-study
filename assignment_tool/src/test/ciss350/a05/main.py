from latextool_basic import *
plot = FunctionPlot(width="3in", height="2in")
plot.add(((100, 1),
          (200, 2),
          (300, 0.5)),
          line_width='2', color='red', legend='bubblesort')
plot.add(((100, 5),
          (200, 3),
          (300, 7)),
          line_width='2', color='green', legend='bubblesort2')
plot.add(((100, 7),
          (200, 4),
          (300, 5)), line_width='2', color='blue', legend='selectionsort')
print(plot)

