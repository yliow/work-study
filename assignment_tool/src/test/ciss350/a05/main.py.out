
\begin{center}
\begin{tikzpicture}[line width=1]
\begin{axis}[width=3in, height=2in,
             scatter/classes={a={mark=*,draw=black}},
             xlabel={\mbox{}},
             xlabel style={name=xlabel}, 
             ylabel={\mbox{}}, 
             legend style={
                at={(xlabel.south)},
                yshift=-1ex,
                anchor=north,
                legend cell align=left,
                },
        ]
]
\addplot[draw=red, line width=2] coordinates {(100, 1)
(200, 2)
(300, 0.5)};\addlegendentry{bubblesort}\addplot[draw=green, line width=2] coordinates {(100, 5)
(200, 3)
(300, 7)};\addlegendentry{bubblesort2}\addplot[draw=blue, line width=2] coordinates {(100, 7)
(200, 4)
(300, 5)};\addlegendentry{selectionsort}
\end{axis}\end{tikzpicture}\end{center}
